src/assets/styles/helper-classes.scss

```diff
@@ -0,0 +1,87 @@
+@use './typography' as typography;
+@use './sizes' as sizes;
+@use './colors' as colors;
+
+@each $name, $pair in typography.$font-size {
+  .font-#{$name} {
+    font-size: $pair !important;
+  }
+}
+
+@each $name, $pair in typography.$font-weight {
+  .font-#{$name} {
+    font-weight: $pair !important;
+  }
+}
+
+@each $name, $value in sizes.$sizes {
+  .sb-m-#{$name} {
+    margin: $value !important;
+  }
+
+  .sb-mt-#{$name} {
+    margin-top: $value !important;
+  }
+
+  .sb-mr-#{$name} {
+    margin-right: $value !important;
+  }
+
+  .sb-mb-#{$name} {
+    margin-bottom: $value !important;
+  }
+
+  .sb-ml-#{$name} {
+    margin-left: $value !important;
+  }
+
+  .sb-mx-#{$name} {
+    margin-left: $value !important;
+    margin-right: $value !important;
+  }
+
+  .sb-my-#{$name} {
+    margin-bottom: $value !important;
+    margin-top: $value !important;
+  }
+
+  .sb-p-#{$name} {
```

Comment:
Hi @lisilinhart! a doubt, should these `.sb-p*` classes be applying margin or should it be `padding`?

Reply:
good question 😄 I think @lisilinhart  just copied the code but it seems like it's wrong, would it be a good chance to change it maybe? 👀